src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -249,7 +249,7 @@ public IConverterManager ConverterManager
         /// <summary>
         /// Gets a helper object for constructing common binding rules for extensions.
         /// </summary>
-        public BindingFactory BindingFactory
+        internal BindingFactory BindingFactory
```

Comment:
>BindingFactory [](start = 17, length = 14)

Remove from public surface since extensions should be using the binding rules (off ExtensionConfigContext) rather than these interfaces.